### PR TITLE
fix: Also allow unicode characters in string literals in 0.4.11

### PR DESCRIPTION
**Pull request by Xanewok**
The type checker started enforcing valid UTF-8 since 0.4.12, but it was possible to use them in the 0.4.11 as well.

### CODE REVIEW

File: .changeset/healthy-carpets-scream.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@nomicfoundation/slang": patch
+---
+
+Accept unicode in string literals in 0.4.11
```
Comment (OmarTawfik) [inline .changeset/healthy-carpets-scream.md:5]: I think this is already covered by `.changeset/eighty-moons-talk.md`?

File: crates/solidity/inputs/language/src/definition.rs [hunk 2]
```diff
@@ -3794,7 +3794,7 @@ codegen_language_macros::compile!(Language(
                             definitions = [
                                 // Allows unicode characters:
                                 TokenDefinition(
-                                    enabled = Range(from = "0.4.12", till = "0.7.0"),
+                                    enabled = Till("0.7.0"),
```
Comment (OmarTawfik) [inline crates/solidity/inputs/language/src/definition.rs:3797]: i see the two tab "\t" tests are also affected. Is it included in these changes?

**Review by OmarTawfik — APPROVED** [review]

File: crates/solidity/inputs/language/src/definition.rs [hunk 3]
```diff
@@ -3794,7 +3794,7 @@ codegen_language_macros::compile!(Language(
...
                                     scanner = Sequence([
```
Comment (OmarTawfik) [inline crates/solidity/inputs/language/src/definition.rs:3798]: I see the two tab "\t" tests are also affected. I wonder if it is included in these changes?

File: .changeset/healthy-carpets-scream.md (hunk 1 above)
Comment (Xanewok) [reply .changeset/healthy-carpets-scream.md:5]: You're right, we can keep this out, then.

File: crates/solidity/inputs/language/src/definition.rs (hunk 3 above)
Comment (Xanewok) [reply crates/solidity/inputs/language/src/definition.rs:3798]: Yes, the tabs where also valid starting from 0.4.11. Try compiling this:


```
pragma solidity ^0.4.11;
contract A {
	string abc = "tab:	";
}
```

**Review by OmarTawfik — APPROVED** [review]

**Pull request merged**